src/main/resources/application.properties

```diff
@@ -64,6 +64,8 @@ vault.url=${vaultUrl}
 vault.token=${vaultToken}
 vault.timeout=30
 
+#Connector Test service details
```

Comment:
not sure that this adds any value

Reply:
This is just comment for information.

Reply:
Thats the exact point i was trying to make: you don't need to add a comment which doesn't add additional information. After all the variable already s called 'connector test service url'.